src/test/java/org/kohsuke/github/GHCheckRunBuilderTest.java

```diff
@@ -26,17 +26,22 @@
 
 import org.junit.Test;
 
+import java.io.IOException;
 import java.util.Date;
 
 import static org.hamcrest.Matchers.containsString;
 
 @SuppressWarnings("deprecation") // preview
-public class GHCheckRunBuilderTest extends AbstractGitHubWireMockTest {
+public class GHCheckRunBuilderTest extends AbstractGHAppInstallationTest {
+
+    protected GitHub getInstallationGithub() throws IOException {
+        return getAppInstallationWithTokenApp3().getRoot();
+    }
 
     @Test
     public void createCheckRun() throws Exception {
-        GHCheckRun checkRun = gitHub.getRepository("jglick/github-api-test")
-                .createCheckRun("foo", "4a929d464a2fae7ee899ce603250f7dab304bc4b")
+        GHCheckRun checkRun = getInstallationGithub().getRepository("hub4j-test-org/test-checks")
+                .createCheckRun("foo", "89a9ae301e35e667756034fdc933b1fc94f63fc1")
```

Comment:
Is there a reason you needed to update existing tests? Once the mocks are created they should continue to work so long as you are not actually changing the behavior of the existing APIs, which AFAICT you are not.

Reply:
It was to make it easier to update them for others by moving the tests to the test org and an app that others can use I believe